translations/en.json

```diff
@@ -1960,5 +1960,6 @@
     "my-albums": "My albums",
     "images": "images",
     "only-6-images": "You can only add 6 images to your message",
-    "delivered": "Delivered"
+    "delivered": "Delivered",
+    "wallet-connect-not-implemented": "Wallet Connect not implemented"
```

Comment:
is it like only for us or for the release app as well? not sure it should be translated

Reply:
Only for us I think, we need to upgrade and re-enable WC 2.0 for the MVP afaik, so this alert shouldn't see the light. I can remove the translation and use a plain string.